### PR TITLE
add title to html and css

### PR DESCRIPTION
**Pull request by eneben**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+body {
```
Comment (ahohnsen) [inline styles.css]: I highly recommend adding the following at the top of your stylesheet:

```css
* {
  box-sizing: border-box;
}
```

This simple line ensures all elements on your webpage inherit `box-sizing: border-box;`, streamlining layout design and making it easier to manage element sizes across different screen resolutions.

You can find detailed information about this property in the handout of the CSS Basic session.

Absolutely, here's a refined version:

Additionally, incorporating `margin: 0;` alongside `box-sizing: border-box;` for all elements is a widely embraced technique. This ensures uniformity and consistency in the spacing and alignment of elements across your webpage.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  margin: 0;
```
Comment (ahohnsen) [inline styles.css:3]:
```suggestion
```

See above - I recommend to move this property to the `*` selector instead.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  height: 15%;
+  background-color: #fefbe9;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+h1 {
+  margin: 0;
+}
```
Comment (ahohnsen) [inline styles.css]: By applying `margin: 0;` within the `*` selector, you eliminate the need to specify this rule individually for each element. This simplifies your stylesheet and ensures a consistent starting point for the layout of all elements on your page

```suggestion

```

**Review by ahohnsen — CHANGES_REQUESTED** [review]
Well done. Please, have a look at my comments. :)

File: styles.css (hunk 1 above)
Comment (eneben) [reply styles.css]: thank you, i will change this.

**Review by ahohnsen — APPROVED** [review]
Very good 💪🏼

**Pull request merged**